lib/enums/query/query_presets.py

```diff
@@ -1,9 +1,14 @@
-from enum import Enum, auto
+from enum import auto
+from enums.query.enum_with_aliases import EnumWithAliases
 from exceptions.parse_query_error import ParseQueryError
 
+# pylint: disable=too-few-public-methods
 
-class QueryPresets(Enum):
-    pass
+
+class QueryPresets(EnumWithAliases):
+    @staticmethod
+    def _get_aliases():
+        pass
```

Comment:
Do we need this? I would argue that this should just be a typing alias: https://stackoverflow.com/questions/33045222/how-do-you-alias-a-type-in-python

```
QueryPresets = EnumWithAliases
```

Reply:
EnumWithAliases is a base class which contains the class method to_string()